### ISSUE TITLE
`PTH201` does not detect `Path("")`

### ISSUE THREAD

**Issue by dosisod** [astral-sh/ruff#6085]
The following 3 snippets are the equivalent, though `Path("")` is not detected:

```python
from pathlib import Path

Path()     # correct code
Path("")   # no error
Path(".")  # PTH201
```

Ruff output:

```
$ ruff x.py
x.py:5:6: PTH201 [*] Do not pass the current directory explicitly to `Path`
```

Expected output:

Ruff should emit an error for `Path("")`. This is a fairly common idiom according to [grep.app](https://grep.app/search?q=%5B%5Ea-zA-Z%5DPath%5C%28%28%22%22%7C%27%27%29%5C%29&regexp=true&case=true&filter[lang][0]=Python), and is currently detected in Refurb.

**Comment by harupy** [comment]
Can I fix this?

**Comment by charliermarsh** [comment]
Go for it :)
